test/SlowTests/Authentication/AuthenticationLetsEncryptTests.cs

```diff
@@ -61,6 +63,23 @@ public async Task CanGetLetsEncryptCertificateAndRenewIt()
             await RenewCertificate(serverCert, firstServerCertThumbprint);
         }
 
+        private static async Task RemoveAcmeCache(string acmeUrl)
+        {
+            var home = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData,
+                Environment.SpecialFolderOption.Create);
+            var caches = Directory.GetFiles(home, "*.lets-encrypt.cache.json");
```

Comment:
don't you need just to do this:
```
            var home = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData,
                Environment.SpecialFolderOption.Create);

            var hash = SHA256.Create().ComputeHash(Encoding.UTF8.GetBytes(url));
            var file = Jws.Base64UrlEncoded(hash) + ".lets-encrypt.cache.json";
```

Would expect some internal static method in the client to calculate this used in LetsEncryptClient ctor and here.
Then you do not need to actually read the file, just remove it?